### ISSUE TITLE
The application "p5" can't be opened

### ISSUE THREAD

**Issue by locksmithdon** [processing/p5.js-editor#32]
I'm using OS X 10.10.1. I downloaded and unzipped `p5.zip` and copied the app to the Applications folder and then tried to run it. That's when I got a dialog with this message. I even tried right-clicking and choosing "Open" since I know default security settings require downloaded apps to be signed. Nope, same message.

So I set off to setup a dev environment. I ran into issues with the first `npm install` because `nodewebkit` was failing to install from its S3 location. After forking this repo and updating `nodewebkit` and `node-webkit-builder` to versions `0.11.5` and `1.0.7` respectively, I got the dev environment setup with everything installed and gulp working. Unfortunately I was then stopped in my tracks by this bug https://github.com/nwjs/nw.js/issues/2518 when trying to `npm run app`. 

I guess I'll just use Sublime for now and try again later. Would love an update if we get this working on Yosemite.

**Comment by marnen** [comment]
Why were you stopped in your tracks? nwjs/nw.js#2518 doesn't prevent the application from running...

**Comment by antiboredom** [comment]
this should be fixed now

**Comment by joesh1** [comment]
Hi, I also have an issue where I download the p5.js from here and when I unzip and then (as above) try to open it nothing happens...nothing at all. Can you maybe advise me (I'm still on Mac OS 10.7.5). I should add I've tried earlier versions of the p5.js installer wondering if I needed an earlier version. So like locksmithdon I guess I'll just run everything from either my Processing editor or use sublime or Komodo.

**Comment by antiboredom** [comment]
Hey would you guys mind trying out some older versions, just to see if this issue is new? It's hard for me to fix cause I can't duplicate. 

https://github.com/processing/p5.js-editor/releases

**Comment by joesh1** [comment]
I've tried out 0.5, I'll go back further and come back to you.

**Comment by antiboredom** [comment]
Thanks! Really appreciate it!

**Comment by joesh1** [comment]
Hi Sam, but I've tried 0.3.0 but no luck. I guess I must be doing something wrong as nobody else seems to be leaving any comments here (or elsewhere). However, I think the lack of clear information and detailed (step-by-step) instructions on the p5 site isn't helping matters. I've tried downloading this complete package + the various editors. The only way I can get something to happen is by opening up  Sublime and pointing it to the famous empty-example folder. But, as Dan Shiffman says, I'll have problems (difficulties) running sound libraries etc using this solution. 

Any help will be brilliant, thanks.
